models/opencontrol.go

```diff
@@ -83,7 +83,7 @@ func (openControl *OpenControl) LoadComponents(directory string) []error {
 	var wg sync.WaitGroup
 	componentsDir, err := ioutil.ReadDir(directory)
 	if err != nil {
-		return []error{ErrReadDir}
+		return []error{errors.New("Error: Unable to read the directory "+directory)}
```

Comment:
We might be able to simply do

```go
return []error{err}
```

I'd do that for the other occurrences of `ErrReadDir` too, and get rid of that error completely for now.